Dockerfile

```diff
@@ -1,5 +1,7 @@
+ARG uid=1000
 FROM debian:buster
 ARG optical_gid
+ARG uid
```

Comment:
Why is the `ARG` repeated here?
Wouldn't it suffice to define it only here like this?

```
ARG uid=1000
```

Reply:
Yes. I have never used build arguments in a docker file before, misread the documentation and have changed this. Thanks.